schema-registry/client/src/main/java/com/hortonworks/registries/schemaregistry/client/SchemaRegistryClient.java

```diff
@@ -228,6 +245,27 @@ public SchemaVersionInfo retrieveSchemaVersion(SchemaIdVersion key) throws Schem
                 .build();
     }
 
+    protected SSLContext createSSLContext(Map<String, String> sslConfigurations) {
+        String defaultPropertyValue = null;
+        SslConfigurator sslConfigurator = SslConfigurator.newInstance();
+        sslConfigurator.keyStoreType(sslConfigurations.getOrDefault("keyStoreType",defaultPropertyValue))
+                .keyStoreFile(sslConfigurations.getOrDefault("keyStorePath",defaultPropertyValue))
```

Comment:
Invoking `sslConfigurations.getOrDefault("keyStorePath",defaultPropertyValue)` with  defaultPropertyValue as null is equivalent to calling `sslConfigurations.get("keyStorePath") ` as that returns null if the value for the given key does not exist as mentioned [here](https://docs.oracle.com/javase/8/docs/api/index.html?java/util/stream/package-summary.html).